torch/_inductor/kernel/flex_attention.py

```diff
@@ -265,24 +263,26 @@ def build_subgraph_buffer(
         # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
         # -- compute scaling constant ---
-        row_max = tl.max(post_mod_scores, 1)
-        m_i_new = tl.maximum(m_i, row_max)
+        m_ij = tl.maximum(m_i, tl.max(post_mod_scores, 1))
```

Comment:
does this actually have an effect on perf?

IMO its more readable with row_max

Reply:
No, I just wanted to mimic the triton tutorial more lol.